tree-sitter/src/TreeSitter/GenerateSyntax.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE DataKinds #-}
 {-# LANGUAGE DeriveAnyClass, DeriveGeneric, LambdaCase, TemplateHaskell, TypeApplications #-}
```

Comment:
Can you do a stylish-haskell pass here (or just make these pragma styles consistent)?